### DISCUSSION TITLE
Your software was Highlighted by JayzTwoCents

### DISCUSSION THREAD

**Discussion by RaVige0** [Rem0o/FanControl.Releases#616] [General]
JayzTwoCents on YouTube highlighted your software on his channel today. Gave it amazing praise and recommended it to his 3.6 million subscribers! Keep up the great work!

https://youtu.be/uDPKVKBMQU8

**Comment by melcarthus** [comment]
YES! i searched for software before on github. never saw this before. Jay linked it in the description. Im going to start using this on my pc. I also donated. This is a real game changer. i just replaced 3 pieces of software that where interfering with each other which is now solved!

**Comment by Mandoon14** [comment]
Just watched that video. This software in its current state is astoundingly wonderful. I praise you Rem0o as a single developer for showing how the big boys should do it!

I use ICue and although good, it has many, many downfalls and issues and of course, is limited, to say the least, and has nowhere near as much ease of use (and potential) as this software given away for FREE.

If in your spare time Rem0o, you could potentially add RGB adjustments for Ram and AIO pumps etc like ICue, but also for different brands that would make me use this software exclusively (ESPECIALLY if you could add an equalizer that also allows you to change from Stereo to auto repositioning and 7.1Ch Surround sound on headphones (I use the Corsair VOID Elite Wireless headphones, probably asking way too much it but would allow me to scrap ICue entirely, which I would love to do).

I intend to donate on payday, and sir (Rem0o), just keep up the great work! I hope an influx of donations heads your way and inspires you to make this already amazing piece of software even better. You rock and we need more people like you.

Kindest regards,
Giles

**Reply by Jonathanese** [reply]
You use iCue? If so, do you use FanControl for the AIO?

I have a 115i and a Commander Pro, and would much prefer to use FanControl over iCue. But I can't get FanControl to see the Commander Pro.

**Reply by Mandoon14** [reply]
Sadly same, here. You would need to connect the fans on your radiator to 4-pin (assuming PWM is used) motherboard fan headers, and then it works. Perhaps this software will one day be updated and allow for fans connected to the pump on an AIO watercooler (mine being the H150i as an example like the 115i you rock) to be detected and override iCue.

PS @Rem0o: My watercooler is powered by a SATA power connector but more importantly connects to a USB 2 header on the motherboard which I'm sure is how iCue reads and changes the fans which are connected to the pump itself, the pump is picked up and read by connecting to the CPU fan pins or like me (if you have one) the pump pins/connector (it's a 4-Pin connection).

If possible maybe when you have some spare time you wouldn't mind making it able to control fans connected that way.

Anyway, take care.

**Reply by Fr0stX76** [reply]
See here https://github.com/Rem0o/FanControl.Releases/discussions/602

**Reply by Y4m4k** [reply]
This is what I use for unifying RGB. You can control virtually all devices regardless of brand.

OpenRGB: this will captured all your devices - https://gitlab.com/CalcProgrammer1/OpenRGB

Artemis: this will read read from OpenRGB - https://artemis-rgb.com/
You will be able to create your RGB setup and customize the lighting however you like. The only catch is there aren’t lighting setups (i.e. it only comes with rainbow and flickering setup). However, if you have some experience with video editing, it has a similar feeling to that for setup your own lighting. It is actually really cool when you get the hang of it.

**Reply by Rem0o** [reply]
^ This. Can't believe Jay didn't mention it. I hope this gets picked up the same way.

**Comment by HuguesGauthier** [comment]
I was not search for this kind of app, Jay linked it in the description. Im going to start using this on my pc. I also donated.
If jay says it's good, so i'm use it  :)

Congrats for your app.
